sdk/communication/azure-communication-mediacomposition/src/main/java/com/azure/communication/mediacomposition/App.java

```diff
@@ -0,0 +1,75 @@
+package com.azure.communication.mediacomposition;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.azure.communication.mediacomposition.implementation.AzureCommunicationMediaCompositionServiceImpl;
+import com.azure.communication.mediacomposition.implementation.AzureCommunicationMediaCompositionServiceImplBuilder;
+import com.azure.communication.mediacomposition.implementation.MediaCompositionsImpl;
+import com.azure.communication.mediacomposition.implementation.models.CommunicationUserIdentifierModel;
+import com.azure.communication.mediacomposition.implementation.models.Layout;
+import com.azure.communication.mediacomposition.implementation.models.LayoutType;
+import com.azure.communication.mediacomposition.implementation.models.MediaCompositionBody;
+import com.azure.communication.mediacomposition.implementation.models.MediaInput;
+import com.azure.communication.mediacomposition.implementation.models.MediaOutput;
+import com.azure.communication.mediacomposition.implementation.models.MediaType;
+import com.azure.communication.mediacomposition.implementation.models.Resolution;
+import com.azure.communication.mediacomposition.implementation.models.Source;
+import com.azure.communication.mediacomposition.implementation.models.SourceType;
+import com.azure.communication.mediacomposition.implementation.models.TeamsMeeting;
+import com.azure.core.http.HttpPipelineBuilder;
+
+public class App
```

Comment:
As you had suggested elsewhere, let's move this under the samples folder. No need to hook up the sample building/running thing yet.